templates/csharp/ai-bot/appsettings.Development.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
```

Comment:
Tip: change this to "Information" so that *Microsoft.TeamsAI* logs can be shown as well.

Reply:
Done.